department-of-veterans-affairs/caseflow
# Queue | Set cases to in_progress on enter Case Details

Resolves #6788
Connects / partially addresses #5800 

### Description
When a user clicks on the CaseDetailsLink for an AMA task, we set its status to `in_progress`. This PR also bolds links to `assigned` tasks for colocated users.

### Acceptance Criteria
- [ ] For colocated users, `assigned` cases' links are bolded
- [ ] Clicking on case details link sets task status to `in_progress`

### Testing Plan
1. Go to `/queue` as `BVALSPORER`, confirm `assigned` tasks are bolded (you may have to `rake db:seed`)
2. Click on case details link, confirm `PATCH` request is successful
3. Return to / reload `/queue`, confirm case details link no longer bolded

<details><summary>gif</summary>
<img src="https://user-images.githubusercontent.com/8533004/45237194-030b6e80-b2ac-11e8-98fb-0f5bc06e380b.gif">
</details>
